apps/src/templates/sectionProgressV2/ProgressFeedbackBanner.jsx

```diff
@@ -46,14 +47,18 @@ const ProgressFeedbackBanner = ({
       bannerStatus === BANNER_STATUS.UNSET &&
       progressV2Feedback
     ) {
-      if (progressV2Feedback.empty) {
+      if (
+        progressV2Feedback.empty &&
+        new Date(userCreatedAt) < new Date('2024-08-02')
```

Comment:
Let's pull this constant into a variable and add a comment to explain why we need it